.github/workflows/cargo_bloat.yml

```diff
@@ -0,0 +1,83 @@
+name: Binary size report
+on:
+  push:
+    branches: [ develop ]
+    paths:
+      - 'libraries/**/*.rs'
+      - 'src/**/*.rs'
+      - 'patches/**'
+      - '**/Cargo.toml'
+      - '.cargo/config'
+      - '!third_party/**'
+  pull_request:
+    branches: [ develop ]
+    types: [opened, synchronize, reopened]
+
+jobs:
+  cargo_bloat:
+    runs-on: ubuntu-18.04
+    steps:
+      # First run: PR
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
+      - name: Install Rust toolchain
+        run: rustup show
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pip setuptools wheel
+      - name: Set up OpenSK
+        run: ./setup.sh
+      - uses: actions-rs/cargo@v1
+        with:
+          command: install
```

Comment:
This means we install with the outdated toolchain. I guess it works but we might want to use stable or nightly. I'm fine keeping like that and fixing later in case of issue.

Reply:
Moved to front, should be checkout independent now.